_harp/about/_company.md

```diff
@@ -1,11 +1,19 @@
 # About us
 
-Founders & Coders C.I.C. runs the only free full-time coding course in the UK. We have developed a highly effective peer-led model of learning which, after only two years of operation, attracts exceptional women and men from all over the world. 30/31 of our most recent graduates are working as software developers.
+Founders & Coders runs a well-established coding camp in London. We are delighted to announce that in 2017 we will be opening our first international campus in the Arab Israeli city of Nazareth.
 
-After completing the course, students are encouraged to join our freelance community and to contribute to a growing array of community activities, including the course itself.
+We follow a highly-effective peer-led approach to learning and we do not employ any teaching or administrative staff. Our low cost of operation allows us to offer our training without charging any fees to our students.
 
-We support ourselves entirely through voluntary contributions from our alumni, through placement fees from employers, and through our close working relationship with [dwyl](http://www.dwyl.io/), which provides freelance experience to many of our graduates.
+The programme is project-based. Starting with a 10-week foundation course, our students then go on to work with charities and early-stage startups to find creative technical solutions to real-world problems.
+
+After students have completed four months of training, they continue to work with us, usually for at least two months, on a variety of projects provided through our partners, while mentoring the next cohort of students.
```

Comment:
Important to mention here they are only part of the teaching staff and are supported by a plethora of more senior members of the teaching 'staff'.